storage/storage.go

```diff
@@ -73,9 +92,16 @@ type Store interface {
 	GetAllStats(ctx context.Context, queues []string) (*Stats, error)
 
 	// Heartbeat
+	GetActiveHeartbeatIDs(ctx context.Context) ([]string, error)
+	GetHeartbeat(ctx context.Context, heartbeatID string) (*Heartbeat, error)
 	SendHeartbeat(ctx context.Context, heartbeat *Heartbeat) error
-	RemoveHeartbeat(ctx context.Context, heartbeat *Heartbeat) error
+	RemoveHeartbeat(ctx context.Context, heartbeatID string) error
+	HandleExpiredHeartbeatIdentities(ctx context.Context) ([]string, error)
+	HandleExpiredWorkerHeartbeats(ctx context.Context, expireTS int64) ([]*StaleMessageUpdate, error)
```

Comment:
How about using the manager heartbeat for this? And not rely on a worker heartbeat?

Reply:
Updated in 75328f44.

Reply:
Thank you for updating this.